src/KerosApp.php

```diff
@@ -43,6 +44,11 @@ public function __construct()
                 $this->get('/{id:[0-9]+}', CatController::class . ':getCat');
                 $this->post("", CatController::class . ':createCat');
             });
+            $this->group('/ua/firm-type', function () {
```

Comment:
Le groupe c'est "UA", le reste est à l'intéieur
Donc + ou - : 
$this->group("/ua")....
...
$this->get("firm-type",...
$this->get("firm-type/id[...]",...